lib/assisted_workflow/addons/github.rb

```diff
@@ -22,12 +22,8 @@ def description
       @issue.body.to_s.gsub("\r\n", "\n")
     end
     
-    def other_id
```

Comment:
to be honest, looking at the code I can't even remember why this other_is is there :cry:
sounds likes it's not been used anywhere